docs/changelog.md

```diff
@@ -5,7 +5,28 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [0.30.0](https://pypi.org/project/sematic/0.30.0/)
+    * [feature] Grafana dashboards tailored for Sematic installable via Helm
+    * [feature] User metrics SDK
+    * [feature] Add horizontal pod autoscaling and pod disruption budget support
+    * [feature] Health endpoint to display DB health in dashboard
+    * [feature] Add support for set types
+    * [feature] Search for runs in dashboard
+    * [feature] Add GitHub commit check support
+    * [feature] Search by tags
+    * [improvement] Endpoint to list users
+    * [improvement] Have resolver continue in in a new resolution if resolver restarts
+    * [improvement] Log request IDs in server
+    * [improvement] Better linting
+    * [improvement] Early resolution failure is more robust
+    * [improvement] Make API retries more robust
+    * [improvement] Enable run search deep links
     * [deprecation] `Calculator` renamed to `Function`
+    * [bugfix] Fixed a DAG view display issue
+    * [bugfix] Fixed application logs being duplicated
+    * [bugfix] Adjust the Name column width distribution
+    * [bugfix] Add missing call to init to Function
+    * [bugfix] Fixed broken link for Plotly docs
```

Comment:
We typically don't mention docs changes in release notes, since the docs are always served from `main` and we don't have any way to "go back in time" to docs at a particular version.